src/report/report-list.controller.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+(function() {
+
+    'use strict';
+
+    /**
+     * @ngdoc controller
+     * @name report.controller:ReportListController
+     *
+     * @description
+     * Controller for report list view page.
+     */
+    angular
+        .module('report')
+        .controller('ReportListController', controller);
+
+    controller.$inject = ['$state', 'reports', 'supersetReports', 'permissions', 'REPORT_RIGHTS'];
+
+    function controller($state, reports, supersetReports, permissions,
+                        REPORT_RIGHTS) {
+        var vm = this;
+        vm.hasRight = hasRight;
+
+        // MW-1207: Adjust reports table view to support categories
+        const LMISReports = reports.filter((report) => {
```

Comment:
I think that we should follow the convention and the variable should be named 'lmisReports'